javalanglib/src/main/scala/java/lang/Long.scala

```diff
@@ -232,4 +233,22 @@ object Long {
     else if (mp != 0) Integer.toOctalString(mp) + padOctal10(lp)
     else Integer.toOctalString(lp)
   }
+
+  /**
+   * Holds the maximal number of digits for each radix, so that a number of that radix
+   * can be converted to an Integer
+   */
+  private def maxExpForRadix(radix: Int): Int = {
```

Comment:
Why is the val named `DigitFitInInt` and this method `maxExpForRadix`? They should probably have related names.